apis/fluent-en_GB/extensions/kotlin_1_3/atrium-api-fluent-en_GB-kotlin_1_3-common/src/main/kotlin/ch/tutteli/atrium/api/fluent/en_GB/kotlin_1_3/resultAssertions.kt

```diff
@@ -26,3 +26,29 @@ fun <E, T : Result<E>> Expect<T>.isSuccess(): Expect<E> = ExpectImpl.result.isSu
  */
 fun <E, T : Result<E>> Expect<T>.isSuccess(assertionCreator: Expect<E>.() -> Unit): Expect<T> =
     ExpectImpl.result.isSuccess(this).addToInitial(assertionCreator)
+
+/**
+ * Expects that the subject of the assertion (a [Result]) is a Failure and
+ * that it encapsulates an exception of type [TExpected].
+ *
+ * @return An assertion container with the new type [TExpected]
+ * @throws AssertionError Might throw an [AssertionError] if the assertion made is not correct.
+ *
+ * @since 0.9.0
+ */
+inline fun <reified TExpected : Throwable> Expect<out Result<Any?>>.isFailure(): Expect<TExpected> =
```

Comment:
please use `Result<*>` instead of `Result<Any?>` is the same but says more about the intend: doesn't matter what value type Result has (same below)